src/pool/srv_pool.c

```diff
@@ -4621,19 +4633,23 @@ ds_pool_evict_handler(crt_rpc_t *rpc)
 }
 
 /**
- * Send a CaRT message to the pool svc during pool destroy to test and
- * (if applicable based on force option) evict all open handles on a pool.
+ * Send a CaRT message to the pool svc to test and
+ * (if applicable based on destoy and force option) evict all open handles
+ * on a pool.
  *
  * \param[in]	pool_uuid	UUID of the pool
  * \param[in]	ranks		Pool service replicas
```

Comment:
the handles and n_handles arguments need to be added to the description of the function

Reply:
In addition to documenting handles/n_handles, suggest force parameter description indicate it is applicable only for destroy=true.